src/gatling/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/gatling/simulations/BookingsReportGenerationTimeSimulation.kt

```diff
@@ -127,6 +127,19 @@ class BookingsReportGenerationTimeSimulation : Simulation() {
   )
     .pause(1.seconds.toJavaDuration())
 
+  private val bookingSearch = exec(
```

Comment:
What are your thoughts around the existing name of this simulation? We've got BookingsReportGenerationTimeSimulation which implies it's only used for reports, which isn't the case with the need to test booking search. 

I suppose this file is already doing a lot of booking set up that we'd like to reuse. I think we could either rename this file to make it more generic, BookingsTimeSimulation? Or make a new one alongside it that extracts and reuses the same booking creation parts?

Reply:
I agree the name isn't really reflective and undersells how much the test is doing. I've renamed it to the proposed BookingsTimeSimulation as that's far more reflective of what it does.

Regarding splitting the test. I think there's a trade of here between cohesive tests and having tests that won't take forever to run in circle. My instinct is to go for fewer large tests, or possibly even one simulation that hits on all areas of interest concurrently. Maybe we break our simulations in to re-usable steps, and then have a CI-specific simulation that makes use of all the parts (if that's at all manageable!)